### PR TITLE
Feature/frontend

### LINKED ISSUE #13: Required Input Validation
Make inputs required for the following fields:
- Date
- Time (Start + End)
- Location
- Weather
- Pilot ID
- Remote ID

### PR DESCRIPTION
**Pull request by Malizma333**
Made form inputs required and added highlighting

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -84,9 +75,9 @@
                 display: flex;
                 align-items: center;
             }
-            ul.nav li {
-               /* float: left; */
-            }
+            /*ul.nav li {
+               	float: left;
+            }*/
```
Comment (cjhr95) [inline src/app.html]: If this is commented out, does it need to stay in the file for any reason?

**Review by cjhr95 — CHANGES_REQUESTED** [review]
There are a lot of similar changes here from the backend PR, so merge that one in and ensure there aren't any merge conflicts

File: src/app.html (hunk 1 above)
Comment (Malizma333) [reply src/app.html]: i don't believe so

**Comment by Malizma333** [comment]
The backend branch has been merged into frontend now

**Review by cjhr95 — APPROVED** [review]
Sweet, looks good

**Pull request merged**